src/SDK/Client/Platform/methods/names/get.ts

```diff
@@ -16,7 +16,7 @@ export async function get(this: Platform, id: string): Promise<any> {
     try{
         const documents = await this.documents.get('dpns.domain', queryOpts);
         return (documents[0] !== undefined) ? documents[0] : null;
-    }catch (e) {
+    } catch (e) {
```

Comment:
Does this `try/catch` make sense?

Reply:
Absolutely not ! 
Removing that non-sense right away.

Reply:
Found 3 instances that I removed